pkg/blob/nodeserver.go

```diff
@@ -260,6 +261,10 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		}
 	}
 
+	if protocol == "" {
+		protocol = Fuse
```

Comment:
I think if protocol is empty, then keep empty value, and empty value should be regarded as fusev1 in the mount finally

Reply:
OK